cmd/ctr/utils_unix.go

```diff
@@ -21,8 +21,8 @@ import (
 	"google.golang.org/grpc/grpclog"
 )
 
-func prepareStdio(stdin, stdout, stderr string, console bool) (*sync.WaitGroup, error) {
-	var wg sync.WaitGroup
+func prepareStdio(stdin, stdout, stderr string, console bool) (wg *sync.WaitGroup, err error) {
+	wg = &sync.WaitGroup{}
```

Comment:
Part of the point of this change is that we no longer need to be careful about the use of `:=`, but I could make the change if you prefer it stylistically.